src/hotspot/share/opto/node.hpp

```diff
@@ -1712,6 +1729,33 @@ class Unique_Node_List : public Node_List {
 #endif
 };
 
+// Definition must appear after complete type definition of Unique_Node_List
+template <typename Callback, typename Check>
+void Node::visit_uses(Callback callback, Check is_boundary) {
+  ResourceMark rm;
+  // Unique_Node_List guarantees no duplicates in the worklist. Note that we
+  // never pop anything from the worklist, since that could result in applying
+  // the callback for a use more than once (if it is at some point readded to
+  // the worklist).
```

Comment:
This is block style, but it would be ok to make lines longer

Reply:
What's the convention? I'm used to aligning so that no line breaks 80 columns, and it looks like most comments in `node.hpp` follow this convention as well.